app/services/notification_service.rb

```diff
@@ -38,6 +38,7 @@ def self.send_for_follows(follows, entity, event_type, event)
         when TOPIC_ADDED_TO_MAP
           next unless TopicPolicy.new(follow.user, entity).show? && MapPolicy.new(follow.user, event.map).show?
           next if follow.user.has_map_open(event.map)
+          next if event.map.source_id # if map is a fork, send notification for that instead
         when TOPIC_CONNECTED_1, TOPIC_CONNECTED_2
```

Comment:
LGTM. We could consider an is_fork? function in thr future but this seems ok

Reply:
Hmm. Just realized that even after the initial forking action, notifications related to topics being added to this map will be squashed due to this line of code. I'll make a note of it in the code and go ahead with it.